src/main/kotlin/com/openlattice/chronicle/controllers/v2/ChronicleControllerV2.kt

```diff
@@ -57,12 +62,12 @@ class ChronicleControllerV2 : ChronicleApi {
             @PathVariable(ChronicleApi.DATASOURCE_ID) datasourceId: String,
             @RequestBody datasource: Optional<SourceDevice>
     ): UUID {
-        if( datasource.isPresent ) {
+        if (datasource.isPresent) {
```

Comment:
Use the default kotlin formatting.

Reply:
Default kotlin formatting doesn't seem to have spaces within the if statement, unless I'm missing something
<img width="1186" alt="Screen Shot 2022-02-08 at 3 45 00 PM" src="https://user-images.githubusercontent.com/15723547/153095605-19d41e11-3631-4453-a879-78015d415a6e.png">

Reply:
Also default tab size seems to be 4

Reply:
Weird I will double check my settings but if formatting is using kotlin coding conventions then we're good.